app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -1242,7 +1247,8 @@ class HomeFragment : Fragment() {
     }
 
     private fun shouldEnableWallpaper() =
-        FeatureFlags.showWallpapers && !(activity as HomeActivity).themeManager.currentTheme.isPrivate
+        FeatureFlags.showWallpapers &&
+            (activity as? HomeActivity)?.themeManager?.currentTheme?.isPrivate?.not() ?: false
```

Comment:
This change was required in order to allow the `HomeFragmentTest::WHEN configuration changed menu is dismissed` test to pass, since that method this is now called on configuration change, and the activity cannot be cast to a `HomeActivity`.